link_test.go

```diff
@@ -1899,9 +1899,9 @@ func TestLinkAltName(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	err = LinkAddAltName(link, "altname")
+	err = LinkAddAltName(link, "some_longer_altname")
```

Comment:
Instead of adding the test code for three alt names can you put them in a slice and have a for loop to test each?

Reply:
Are you fine with the new aproach?

Reply:
Yes, thank you.
Can you pls take care of few other very minor comments I added, and most importantly squash your commits into one.
This project follows a "one commit per feature/fix" rule.
Thank you